mlrun/config.py

```diff
@@ -268,6 +268,17 @@
             # - mlrun.runtimes.function.enrich_function_with_ingress
             "add_templated_ingress_host_mode": "never",
         },
+        "pipelines": {
+            "logs": {
+                "pull_state": {
+                    "mode": "disabled",
+                    # those params are used when mode is enabled
+                    "pull_logs_interval": 30,  # seconds
+                    "pull_state_interval": 5,  # seconds
+                },
+                "pull_logs_default_interval": 3,  # seconds
+            },
+        },
```

Comment:
It isn't clear enough what is the difference between `pull_logs_interval` and `pull_logs_default_interval`. I'm also not sure I found in the PR the use of `pull_logs_default_interval`. Please add a comment explaining the difference and the use of each config variable.